subsys/net/lib/openthread/platform/logging.c

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
A convention for OT related commits is to prefix the tile with `net: openthread: ...`, please update it while rebasing.

Reply:
changed commit message